bin/varnishd/mgt/mgt_child.c

```diff
@@ -603,7 +603,7 @@ MGT_Child_Cli_Fail(void)
 		return;
 	if (kill_child() == 0)
 		MGT_complain(C_ERR, "Child (%jd) not responding to CLI,"
-		    " killing it.", (intmax_t)child_pid);
+		    " killed it.", (intmax_t)child_pid);
```

Comment:
Sorry, you were a tad too late, Dridi. But if the end result (ie. the code) is ok, then I am happy.